### PR TITLE
add loging panel and bootstrap styling

### PR DESCRIPTION
**Pull request by Maciorex**

### CODE REVIEW

File: app/controllers/home_page_controller.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class HomePageController < ActionController::Base
+class HomePageController < ApplicationController
     def index
```
Comment (ozimeu) [inline app/controllers/home_page_controller.rb]: this method has empty body, should be an one-liner

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -45,10 +26,10 @@
...
           <span class="icon-bar"></span>
         </button>
-        <a class="navbar-brand" href="#">TrainHard</a>
+        <a class="navbar-brand" href="#">StockTracker</a>
         <div class="navbar-collapse collapse navbar-responsive-collapse">
           <ul class="nav navbar-nav">
-            <li><%= link_to "Link1", "/path1"  %></li>
+            <li><%= link_to "Jajo", "/path1"  %></li>
             <li><%= link_to "Link2", "/path2"  %></li>
             <li><%= link_to "Link3", "/path3"  %></li>
```
Comment (ozimeu) [inline app/views/layouts/application.html.erb]: this links here seems not to be valid, try not to commit broken code. Please add routes to handle this or change the paths to `#` or something that won't cause app error

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -4,30 +4,11 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title><%= content_for?(:title) ? yield(:title) : "TrainHard" %></title>
+    <title><%= content_for?(:title) ? yield(:title) : "StockTracker" %></title>
```
Comment (ozimeu) [inline app/views/layouts/application.html.erb]: we should avoid putting logic in the views, consider moving this into helper/decorator

File: public/assets/.sprockets-manifest-8555c949cd6592490c70cbd0375ea8d7.json [hunk 4]
```diff
@@ -0,0 +1 @@
...ea9361bdcf0b442d06a18a8051af57456","integrity":"sha256-45UEQJN1fYKvyxOJV9BqHqk2G9zwtELQahioBRr1dFY="},"glyphicons-halflings-regular-42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5.svg":{"logical_path":"glyphicons-halflings-regular.svg","mtime":"2018-02-22T16:24:05+01:00","size":108738,"digest":"42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5","integrity":"sha256-QvYGWdJlwaPDD5+kKry7Vr1KU69Ng9MW1t16NpA8Q+U="}},"assets":{"application.js":"application-28fa8e3f5a442dcf5ed50188edad6735f09b92339a7c7e77a8c089b5f0f87703.js","application.css":"application-c3edde78272ef25d9c3ea3af466e8e0b8f48861e44dedd2a11703f9942d16614.css","glyphicons-halflings-regular.eot":"glyphicons-halflings-regular-13634da87d9e23f8c3ed9108ce1724d183a39ad072e73e1b3d8cbf646d2d0407.eot","glyphicons-halflings-regular.woff":"glyphicons-halflings-regular-a26394f7ede100ca118eff2eda08596275a9839b959c226e15439557a5a80742.woff","glyphicons-halflings-regular.ttf":"glyphicons-halflings-regular-e395044093757d82afcb138957d06a1ea9361bdcf0b442d06a18a8051af57456.ttf","glyphicons-halflings-regular.svg":"glyphicons-halflings-regular-42f60659d265c1a3c30f9fa42abcbb56bd4a53af4d83d316d6dd7a36903c43e5.svg"}}
```
Comment (ozimeu) [inline public/assets/.sprockets-manifest-8555c949cd6592490c70cbd0375ea8d7.json:1]: I don't think these assets files should be included in this commit. You can leave it here for now, but it's priceless to keep your commits atomic.

**Review by ozimeu — CHANGES_REQUESTED** [review]
please resolve the issues pointed out in the comments

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (Maciorex) [reply app/views/layouts/application.html.erb]: corrected!

File: app/controllers/home_page_controller.rb (hunk 1 above)
Comment (Maciorex) [reply app/controllers/home_page_controller.rb]: now it is one-liner!

**Review by ozimeu — APPROVED** [review]
:+1:

**Pull request merged**